woocommerce/woocommerce
# [Experimental] Product Filter: Price (Beta) block - Disable block interactions within the editor

### Changes proposed in this Pull Request:
This PR disables the user interactions with the Product Filter: Price (Beta) block within the Editor.





Closes #45361 .



### How to test the changes in this Pull Request:



Using the [WooCommerce Testing Instructions Guide](https://github.com/woocommerce/woocommerce/wiki/Writing-high-quality-testing-instructions), include your detailed testing instructions:

1. Navigate to Single Product template (Appearance > Editor > Templates > Product Catalog template - or any archive template).
2. Click on "Edit" to open the WordPress editor.
3. In the editor, you should see a block inserter icon or an option to "Add Block". Click on it.
4. Search for "Product Filter: Price (Beta)" in the block inserter or browse the available blocks until you find them. Add it to your content.
5. Make sure you cannot interact with the input fields or the price slider of the block within the Editor.

#### Before

https://github.com/woocommerce/woocommerce/assets/20469356/a58f4235-59cb-48e4-94db-0d12b3e39401

#### After

https://github.com/woocommerce/woocommerce/assets/20469356/bc79c0f3-4801-436f-9b35-29b31c0b6646



### Changelog entry




<details>

#### Significance




#### Type




#### Message 
Prevent user interaction with the Product Filter: Price (Beta) block within the Editor.

#### Comment 

</details>
